net/ddns-scripts/files/usr/lib/ddns/update_pdns.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+# Derived from update_gandi_net.sh
+. /usr/share/libubox/jshn.sh
+
+local __TTL=600
+local __RRTYPE
+local __STATUS
+
+[ -z "$username" ] && write_log 14 "Service section not configured correctly! Missing subdomain as 'username'"
+[ -z "$password" ] && write_log 14 "Service section not configured correctly! Missing API Key as 'password'"
+[ -z "$param_opt" ] && write_log 14 "Service section not configured correctly! Missing PowerDNS URL as 'Optional Parameter'(param_opt)"
+
+# Create endpoint from $param_opt
+# e.g. param_opt=http://127.0.0.1:8081
+local __ENDPOINT="$param_opt/api/v1/servers/localhost/zones"
+# Build JSON payload
+json_init
+json_add_object rrsets
+	json_add_string name "$username.$domain"
+	json_add_string type "$__RRTYPE"
+	json_add_string ttl "$__TTL"
+	json_add_string changetype "REPLACE"
+	json_add_object records
+		json_add_string content "$__IP"
+		json_add_boolean disabled 0
+	json_close_object
+json_close_object
+
+[ $use_ipv6 -ne 0 ] && __RRTYPE="AAAA" || __RRTYPE="A"
```

Comment:
I think this should go before this [line](https://github.com/openwrt/packages/pull/16429/files#diff-db0a3148a664e717a10cfde37eb8b93a90c27e376cdc255b6c79137a205b196eR16). Otherwise it makes no sense?

Also please remove the tabs when you generate the json this is not common